docs/getting-started.md

```diff
@@ -50,13 +60,29 @@ helm repo add antrea https://charts.antrea.io
 helm install antrea antrea/antrea -n kube-system --set featureGates.FlowExporter=true
 ```
 
+Ensure `flowExporter.enable` flag is set to true when the version of Antrea is later than v1.13.
```

Comment:
To me, this paragraph should switch the order with the paragraph above. So the order will be like:

- `>=v1.13`

- `v1.8 - v1.12`

not a strong opinion though